### PR TITLE
[TA] Fixed the failing live tests

### PR DESCRIPTION
**Pull request by mssfang**
Most of the commits are testing the live-tests failure for "invalid endpoint error", which turn out to be the devops variable library had it as `java-azure-textanalytics-sdk-endpoint` but the secret name in keyvault matches what TA have in the yml, `java-textanalytics-sdk-endpoint`.

After resolved it, the actual live test failure occurs, the file changes are apply to the changes.

### CODE REVIEW

**Review by samvaity — APPROVED** [review]

File: sdk/textanalytics/tests.yml [hunk 1]
```diff
@@ -4,13 +4,9 @@ stages:
   - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: textanalytics
+      TimeoutInMinutes: 60
+      MaxParallel: 7
```
[the remark this answers is not in the document]
Comment (mssfang) [reply sdk/textanalytics/tests.yml:8]: Yes. I was testing if it will make live-test fast

File: sdk/textanalytics/tests.yml [hunk 2]
```diff
@@ -12,5 +15,6 @@ stages:
       # Will enable it when service rolls out AAD to all region.
       DisableAzureResourceCreation: true
       EnvVars:
+        AZURE_TEST_MODE: RECORD
```
Comment (alzimmermsft) [inline sdk/textanalytics/tests.yml:18]: You only need to set `TestMode`, setting both `TestMode` and `AZURE_TEST_MODE` will result in an invalid YML configuration

File: sdk/textanalytics/tests.yml [hunk 3]
```diff
@@ -4,6 +4,9 @@ stages:
   - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
     parameters:
       ServiceDirectory: textanalytics
+      TimeoutInMinutes: 60
+      TestMode: RECORD # to make sure recordedData is not null but still testing LIVE mode
+      MaxParallel: 7
```
Comment (alzimmermsft) [inline sdk/textanalytics/tests.yml:9]: We don't need to add `MaxParallel` since we've moved to a new VM system.

**Pull request merged**